cluster/azure/util.sh

```diff
@@ -34,6 +34,9 @@ KUBE_CONFIG_FILE="${KUBE_CONFIG_FILE:-"${DIR}/config-default.sh"}"
 source "${KUBE_CONFIG_FILE}"
 source "${KUBE_ROOT}/cluster/common.sh"
 
+AZKUBE_VERSION="v0.0.4"
+REGISTER_MASTER_KUBELET="true"
```

Comment:
This value is used by `validate-cluster.sh` to check for the number of nodes reported. Relates to whether or not the master kubelet is brought up with `--register-node={true,false}`.